File: server/datasets/java_source-code/Multiple-E2/type1_replace_clone_1_valid_parentheses.java
public class Solution {
    public boolean isValid(String s) {
        int prev_length = -1;
        while (s.length() != prev_length) {
            prev_length = s.length();
            
            s = s.replace("()", "");
            s = s.replace("{}", "");
            s = s.replace("[]", "");
        }
        return s.length() == 0;
    }
}

File: server/datasets/python_source-code/Multiple-E2/type1_replace_clone_1_valid_parentheses.py
def isValid(s):
    prev_length = -1
    while len(s) != prev_length:
        prev_length = len(s)
        
        s = s.replace("()", "")
        s = s.replace("{}", "")
        s = s.replace("[]", "")
        
    return len(s) == 0
